Userguide/Addadevice.md

```diff
@@ -5,7 +5,11 @@ In order to add a device to the platform, call the add device method in the devi
 
 ### Implementation may differ for different device types
 Each device type may require its own set of attributes, security settings, etc.
-Each device type may have its own registration mechanism
+Each device type may have its own registration mechanism.

```

Comment:
Please state that this are exmaples examples. More domain specific information should be added to the chapter with the domains.